src/NosCore.Controllers/Game/GameStartPacketHandler.cs

```diff
@@ -208,18 +208,18 @@ public override async Task Execute(GameStartPacket packet, ClientSession session
 
             //            // finfo - friends info
             var mails = await _mailHttpClient.GetGifts(session.Character.CharacterId);
-            session.Character.GenerateMail(mails);
+            await session.Character.GenerateMail(mails);
 
-            session.SendPacket(session.Character.GenerateTitle());
+            await session.SendPacket(session.Character.GenerateTitle());
             int giftcount = mails.Select(s=>s.MailDto).Count(mail => !mail.IsSenderCopy && mail.ReceiverId == session.Character.CharacterId && mail.ItemInstanceId != null && !mail.IsOpened);
             int mailcount = mails.Select(s => s.MailDto).Count(mail => !mail.IsSenderCopy && mail.ReceiverId == session.Character.CharacterId && mail.ItemInstanceId == null && !mail.IsOpened);
             if (giftcount > 0)
             {
-                session.SendPacket(session.Character.GenerateSay(string.Format(Language.Instance.GetMessageFromKey(LanguageKey.GIFTED, session.Account.Language), giftcount), SayColorType.Purple));
+                await session.SendPacket(session.Character.GenerateSay(string.Format(Language.Instance.GetMessageFromKey(LanguageKey.GIFTED, session.Account.Language), giftcount), SayColorType.Purple));
             }
             if (mailcount > 0)
             {
-                session.SendPacket(session.Character.GenerateSay(string.Format(Language.Instance.GetMessageFromKey(LanguageKey.NEW_MAIL, session.Account.Language), mailcount), SayColorType.Yellow));
+                await session.SendPacket(session.Character.GenerateSay(string.Format(Language.Instance.GetMessageFromKey(LanguageKey.NEW_MAIL, session.Account.Language), mailcount), SayColorType.Yellow));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 204 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)